x/payment/spec/docs.md

```diff
@@ -113,15 +113,15 @@ if err != nil {
 }
 
 // query for account information necessary to sign a valid tx
-err = keyringSigner.QueryAccount(ctx, grpcClientConn)
+err = keyringSigner.QueryAccountNumber(ctx, grpcClientConn)
```

Comment:
Is this the method we should be using?

Reply:
this brings up a really good point. This method appears to be vestigial, and should likely not be exported. Instead, it should be private and only called during the keyringSigner's BroadcastTx method.

Reply:
#294

Reply:
For the moment though, is this a necessary component to building and submitting a PFD?